test/benchmarks_test.dart

```diff
@@ -22,7 +23,7 @@ final valueList = <String>[
 /// Tests that the Gallery web benchmarks are run and reported correctly.
 Future<void> main() async {
   test('Can run a web benchmark', () async {
-    print('Starting web benchmark tests ...');
+    stdout.writeln('Starting web benchmark tests ...');
```

Comment:
These changes seem odd, any idea why they happened?

Reply:
Yeah, this it the lint https://dart-lang.github.io/linter/lints/avoid_print.html